proto/ots/bettor.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package ots;
+
+option go_package = "oddin.gg/ots";
+option java_package = "com.oddin.ots";
+
+import "google/protobuf/timestamp.proto";
+import "ots/commons.proto";
+
+message BettorsRequest {
+  // if recover_since is null, no updates will be recovered. If it is defined we will start to send to consumer
+  // all BettorRiskCoefficients since given timestamp value
+  optional google.protobuf.Timestamp recover_since = 1 ;
+}
+
+message BettorsResponse {
+  oneof data {
+    Bettor bettor = 1;
+    StreamKeepalive keepalive = 2;
+  }
+}
+
+message Bettor {
+  // internal bettor id
+  string id = 1;
+  // external bettor id
+  string ext_id = 2;
```

Comment:
I am not sure if this is the best approach. IMHO, customer should not know about our internal ID. In other messages I can see just this:

```
  // Unique ticket id from operator’s system.
  string id = 1;
```

Reply:
@bednarradek Please enhance the documentation comment according to Radek's comment.

Reply:
✅ done